src/pages/Home/Home.test.js

```diff
@@ -0,0 +1,338 @@
+import React from 'react';
+import 'fake-indexeddb/auto';
+
+import Home from 'pages/Home';
+import { fireEvent, render, screen, waitFor } from 'setupTests';
+
+let emptyStatistics = {
+  totalGames: 0,
+  totalWins: 0,
+  totalAttempts: [0, 0, 0, 0, 0, 0, 0],
+  currentStreak: 0,
+  lastDatePlayed: '',
+  longestStreak: 0,
+  longestStreakDate: '',
+  attemptedWords: {},
+  timeAverage: 0,
+  minTime: 0,
+  maxTime: 0,
+};
+
+jest.mock('components/common/Loading');
+jest.mock('react-confetti');
+
+jest.mock('hooks/useSlackApp', () => () => ({
+  sendMessageToChannel: jest.fn(),
+}));
+
+jest.mock('firebase/words', () => ({
+  getTodaysWord: () => ({ todaysWord: 'GENIE' }),
+}));
+
+jest.mock('firebase/dailyResults', () => ({
+  addDailyResults: () => '1234567',
+  getDailyResults: () => ({
+    docs: [],
+    error: false,
+  }),
+  updateDailyResults: jest.fn(),
+}));
+
+jest.mock('firebase/usersStatistics', () => ({
+  getUsersStatistics: () => ({
+    currentStatistics: emptyStatistics,
+  }),
+  updateUsersStatistics: () => {},
+}));
+
+const getKeyboardLetter = letter => screen.getByRole('button', { name: letter });
+
+const getLetters = buttons =>
+  buttons.filter(item => {
+    const ariaRoleDescription = item.getAttribute('aria-roledescription');
+    return ariaRoleDescription === 'letter';
+  });
+
+beforeEach(async () => {
+  render(<Home />, {
+    preloadedState: {
+      user: {
+        user: {
+          email: 'janedoe@rootstrap.com',
+          name: 'Jane Doe',
+          photo: 'https://source.unsplash.com/random/?person',
+          uid: 'ABCDEFGHIJK',
+        },
+        authenticated: true,
+      },
+    },
+  });
+
+  await waitFor(() => {
+    const emptyWord = screen.getAllByRole('group', { name: 'empty' });
+    expect(emptyWord).toHaveLength(1);
+
+    const emptyLetters = screen.getAllByRole('button', { name: 'empty' });
+    expect(emptyLetters).toHaveLength(5);
+  });
+});
+
+test('enter all incorrect letters (CLASS)', async () => {
+  // get all the keyboard buttons we will use
+  const CButton = getKeyboardLetter('C');
+  const LButton = getKeyboardLetter('L');
+  const AButton = getKeyboardLetter('A');
+  const SButton = getKeyboardLetter('S');
+
+  // type CLASS with the keyboard buttons
+  fireEvent.click(CButton);
+  fireEvent.click(LButton);
+  fireEvent.click(AButton);
+  fireEvent.click(SButton);
+  fireEvent.click(SButton);
+
+  // check there are no more empty spaces and now it has CLASS content
+  const emptyWord = screen.queryByRole('group', { name: 'empty' });
+  expect(emptyWord).not.toBeInTheDocument();
+
+  const classWord = screen.queryByRole('group', { name: 'CLASS' });
```

Comment:
```suggestion
  const classWord = screen.getByRole('group', { name: 'CLASS' });
```
In this case since is expected that the element will be in the document we should use `getByRole`, `queryByRole` is used when you know the element is not going to be in the document (like the `emptyWord` above)